src/Control/Effect/Labelled.hs

```diff
@@ -70,6 +77,35 @@ instance Algebra (eff :+: sig) (sub m) => Algebra (Labelled label eff :+: sig) (
     R sig -> Labelled . alg (runLabelled . hdl) (R sig)
   {-# INLINE alg #-}
 
+type HasLabelledLift n sig m = HasLabelled Lift (Lift n) sig m
+
+newtype LabelledLift (label :: k) (sub :: (Type -> Type) -> (Type -> Type)) m a = LabelledLift (m a)
+  deriving
+    ( Alternative
+    , Applicative
+    , Functor
+    , Monad
+    , Fail.MonadFail
+    , MonadFix -- ^ @since 1.1.1
```

Comment:
No need for the `since` declaration here. However, can you add `since` declarations for `LabeledLift`, `runLabeledLift`, and `lift`?

Reply:
I don't mind the `since` here, FWIW, as long as haddock picks it up properly.

Reply:
Top function specified version @since 1.1.2.2